autogen/agentchat/contrib/multimodal_conversable_agent.py

```diff
@@ -64,44 +60,21 @@ def update_system_message(self, system_message: Union[Dict, List, str]):
         self._oai_system_message[0]["role"] = "system"
 
     @staticmethod
-    def _message_to_dict(message: Union[Dict, List, str]):
-        """Convert a message to a dictionary.
+    def _message_to_dict(message: Union[Dict, List, str]) -> Dict:
+        """Convert a message to a dictionary. This implementation
+        handles the GPT-4V formatting for easier prompts.
 
-        The message can be a string or a dictionary. The string will be put in the "content" field of the new dictionary.
+        The message can be a string or a dictionary. The string will be put in
```

Comment:
a string or a dictionary or a list of dictionaries?

Reply:
Good catch. Resolved and also added more assertions in the function.